.gitlab-ci-darwin.yml

```diff
@@ -0,0 +1,83 @@
+variables:
+  SCHEDULER_PARAMETERS: '--nodes=1 --partition=power9 --export=NONE'
+  GIT_SUBMODULE_STRATEGY: recursive
+  MODULE_CMAKE: "cmake/3.11.1"
+  MODULE_CLANG: "clang/8.0.1"
+  MODULE_COMPILER: "gcc/7.4.0"
+  MODULE_CUDA: "cuda/10.1"
+  MODULE_MPI: "openmpi/p9/4.0.2-gcc_7.4.0"
+  NUM_MPI_PROC_TESTING: 2
+  OMP_NUM_THREADS: 40
+  CMAKE_BUILD_TYPE: "Release" 
+  Kokkos_ARCH_POWER9: "ON" 
+  Kokkos_ARCH_VOLTA70: "ON" 
+  Kokkos_ENABLE_CUDA: "ON" 
+  Kokkos_ENABLE_CUDA_UVM: "OFF"
+  Kokkos_ENABLE_CXX11: "ON" 
```

Comment:
I don't think this is meant to be defined - I think Kokkos sets this internally. We already specify the C++ 14 standard to Kokkos via `Kokkos_CXX_STANDARD`. Do we need this?

Reply:
Yes, this needs to be updated, the standard varaible does need to be accessible though, I think for getting the xl compiler to work. I could be wrong but xl may not work with all features of c++14.

Reply:
Is this potentially related to #256 ?
It somehow feels odd to set CXX11 here while our default in the root CMakeLists.txt is supposed to be 14.

Reply:
> Is this potentially related to #256 ?
> It somehow feels odd to set CXX11 here while our default in the root CMakeLists.txt is supposed to be 14.

Not to my knowledge. I will remove it.